es6-shim.js

```diff
@@ -291,6 +291,13 @@
       return x != null && Object(x) === x;
     },
 
+    TypeIsNotObject: function (val) {
+      // This is faster when TypeIsObject may commonly return false.
+      return val === void 0 || val === null || val === true || val === false ||
+        typeof val === 'string' || typeof val === 'number' ||
+        typeof val === 'symbol'; // jshint ignore:line
```

Comment:
`symbol` is a valid typeof value, so rather than the jshint comment, let's modify the project config to not complain about that.